src/html/loading/index.html

```diff
@@ -7,7 +7,12 @@
   <body>
     <div class="container">
       <div class="loader"></div>
-      <h1>Initializing...</h1>
+      <h1 id='loading-text'>初始化中...</h1>
+      <script>
+        const locale = window.navigator.userLanguage || window.navigator.language;
```

Comment:
@alex-cory will locale be `undefined`? if so, L13 will throw an error. If not, LGTM

Reply:
no those 2 cases should cover all browsers